src/zenml/integrations/integration.py

```diff
@@ -43,6 +43,17 @@ def __new__(
         cls = cast(Type["Integration"], super().__new__(mcs, name, bases, dct))
         if name != "Integration":
             integration_registry.register_integration(cls.NAME, cls)
+
+        if getattr(cls, "REQUIREMENTS"):
+            logger.warning(
+                f"Integration {cls.NAME} has a REQUIREMENTS attribute. "
+                f"Please use the get_requirements() method instead."
+                f"REQUIREMENTS will be deprecated in the future."
+            )
```

Comment:
I would now even suggest to keep the current `REQUIREMENTS` syntax as well (and to use it for airflow and pytorch again).

The reason why I changed my mind on this is that the integrations might move into the Hub in the somewhat-near future at which point we will have to deprecate/adjust all of this anyways. Thus, I'd suggest to not spend too much effort on this PR right now and just go with the minimal viable solution that makes TensorFlow work on Mac.

```suggestion
```

Reply:
I agree, the old way should still be possible without a warning and should simply be used as the default returned by `get_requirements` (which you already implemented here.